### ISSUE TITLE
xbar not showing at all 

### ISSUE THREAD

**Issue by jonathanalevi** [matryer/xbar#835]
On M1 Max MacBook Pro (with notch), xbar isn't showing at all, despite reinstalling etc.

**Comment by rokii666** [comment]
Same here
![Uploading Screenshot 2022-05-14 at 13.02.07.png…]()

**Comment by leaanthony** [comment]
Does the m1 pro have 3x density retina screens?

**Comment by rokii666** [comment]
Oh, I am sorry, I guess I understand your issue not correctly - I thought you have same problem as mine - not showing GAS price, not xbar in general. I do see xbar, but it just shows what on screenshot. I do not know about density, I guess they should be the same.

**Comment by dbuezas** [comment]
same here, intel on 2 different intel macs (Monterey 12.3.1)

**Comment by cunningr** [comment]
I also see some strange behaviour (M1 16in).  I have very basic title strings but some just make the bar disappear.  Although the plugin is still running, it just isn't visible.

works;
```
'a-short-string'.upper()
```

doesn't work;
```
'some-long-string-------25'.upper()
```

Interestingly enough, without the `'.upper()` the longer string works up to 28 chars;
```
some-long-string--------s-28
```

xbar -> `v2.1.7-beta`
MacOS -> `Monterey 12.3.1` (MacPro M1 16)

**Comment by NoComment73** [comment]
Hm, installed xbar for the first time, M1 Max MBPro 16, showed up for some time, was able to setup plugins, but while the plugin window was shown, it disappeared but is still running. Quiting and re-starting don't solve the issue. It runs (Activity Monitor) but it doesn't show.

**Comment by leaanthony** [comment]
Yeah looks like there's some compatibility issue with M1 Max. Need to find one someone is giving away to test 😀

**Comment by huyz** [comment]
> Need to find one someone is giving away to test

Probably too expensive still but this is an option: https://aws.amazon.com/ec2/instance-types/mac/

**Comment by leaanthony** [comment]
Thanks for sharing. Doesn't look like they do Max though 😔

**Comment by huyz** [comment]
> Doesn't look like they do Max though

You think that Max makes any difference? For example, M1 Pro has the same display: https://en.wikipedia.org/wiki/MacBook_Pro_(Apple_silicon)#Technical_specifications_2

**Comment by cunningr** [comment]
Fwiw I seem to see this issue when I have an external wide monitor connected. The issue then seems to be on the Mac screen

**Comment by NoComment73** [comment]
> > Doesn't look like they do Max though
> 
> You think that Max makes any difference? For example, M1 Pro has the same display: https://en.wikipedia.org/wiki/MacBook_Pro_(Apple_silicon)#Technical_specifications_2

AWS uses Mac mini with M1. Can anyone confirm that a Mac mini with M1 has the same issue with xbar?

**Comment by NoComment73** [comment]
If this is a problem on any Mac Silicon, I may be able to provide you with a remote access Mac mini in the cloud. Let me know if this would help.

**Comment by elnjensen** [comment]
> Just playing about with xbar on an 14" M1 MBP (with notch) - gasnow plugin is broken as the URL is defunct(redirects to the generic BitCoin page) but xbar is working OK. I do note however that the notch functions just like a black hole so if the menus to the right of the notch fill up the space then any more icons will be gobbled by the notch. I tried a whole lot of plugins to see if they would appear on the other side of the notch but they don't so it would appear that the space available for menu icons is limited to the right of the notch!
> 
> I guess it may be possible that if the space to the right of the notch is already full then if xbar is opened it will immediately be dispatched to the notch.

Update - this appears to be the case for me, as @Blatman suggested.   Workaround is to remove some of the existing menubar icons, start xbar, and then drag it farther to the right before restoring the other icons (which can successfully avoid the notch).

**Comment by sshanky** [comment]
For me (Mac Mini M1 8GB) my xbar items are visible but nothing happens when I use them. Quitting and restarting solves the problem.

**Comment by leaanthony** [comment]
Is there any output on the console that can suggest what is happening?

**Comment by sshanky** [comment]
How would I do that?

**Comment by deepy** [comment]
I've got a Mac with a notch and if I start xbar it doesn't appear in the bar
If I then launch Bartender 4, it happily appears in Bartender 4
And if I then quit Bartender 4, it shows up in my bar again

This does not happen when the bar is on an external monitor
